static/js/popovers.js

```diff
@@ -696,10 +696,18 @@ export function popover_items_handle_keyboard(key, $items) {
 
     if (key === "enter" && index >= 0 && index < $items.length) {
         $items[index].click();
+        if ($current_user_info_popover_manage_menu) {
+            const $items = get_user_info_popover_manage_menu_items();
+            focus_first_popover_item($items);
+        }
         return;
     }
     if (index === -1) {
-        index = 0;
+        if (message_info_popped() || user_info_popped() || user_sidebar_popped()) {
+            index = 1;
+        } else {
+            index = 0;
+        }
```

Comment:
These index calculations were wrong; if you clicked on a popover that didn't have the new vdots menu (say, your own popover menu), we'd start with the second item incorrectly. The right way to write this check in general is to have the code say what you mean: The index should be increased the `\vdots` element is visible.

```
diff --git a/static/js/popovers.js b/static/js/popovers.js
index 42c68d37e4..424898c636 100644
--- a/static/js/popovers.js
+++ b/static/js/popovers.js
@@ -714,7 +714,7 @@ export function popover_items_handle_keyboard(key, $items) {
         return;
     }
     if (index === -1) {
-        if (message_info_popped() || user_info_popped() || user_sidebar_popped()) {
+        if ($(".user_info_popover_manage_menu_btn").is(":visible")) {
             index = 1;
         } else {
             index = 0;
@@ -847,8 +847,12 @@ function focus_user_info_popover_item() {
     // For now I recommend only calling this when the user opens the menu with a hotkey.
     // Our popup menus act kind of funny when you mix keyboard and mouse.
     const $items = get_user_info_popover_for_message_items();
-    // Focus element at index 1 first, skipping three-dot menu which is on index 0.
-    focus_first_popover_item($items, 1);
+
+    if ($(".user_info_popover_manage_menu_btn").is(":visible")) {
+        focus_first_popover_item($items, 1);
+    } else {
+        focus_first_popover_item($items);
+    }
 }
 
 function get_user_sidebar_popover_items() {
```

There's probably some generalization refactoring we could do to make such a `\vdots` menu a general component, but I'm not going to block this PR on that.